src/lib.rs

```diff
@@ -636,17 +639,20 @@ impl<'conn> SqliteStatement<'conn> {
         Ok(SqliteRows::new(self))
     }
 
-    pub fn query_map<'a, 'map, T, F>(&'a mut self, params: &[&ToSql], f: F)
-                                     -> SqliteResult<MappedRows<'a, 'map, T>>
+    /// Executes the prepared statement and maps a function over the resulting
+    /// rows. 
+    ///
+    /// Unlike the iterator produced by `query`, the returned iterator will
+    /// never panic.
```

Comment:
This claim is too strong - `f` might panic itself, for example. Maybe "Unlike the iterator produced by `query`, the returned iterator does not expose the possibility for accessing stale rows." ?